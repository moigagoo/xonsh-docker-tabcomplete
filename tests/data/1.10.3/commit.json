{
        "arguments": [
                "-a", "--author",
                "-c", "--change=[]",
                "--help",
                "-m", "--message",
                "-p", "--pause=true"
        ],
        "commands" : []
}
