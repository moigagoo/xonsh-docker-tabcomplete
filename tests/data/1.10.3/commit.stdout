
Usage:	docker commit [OPTIONS] CONTAINER [REPOSITORY[:TAG]]

Create a new image from a container's changes

  -a, --author        Author (e.g., "John Hannibal Smith <hannibal@a-team.com>")
  -c, --change=[]     Apply Dockerfile instruction to the created image
  --help              Print usage
  -m, --message       Commit message
  -p, --pause=true    Pause container during commit
